### PR TITLE
oggm ev

### PR DESCRIPTION
**Pull request by fmaussion**
This adds a page for the organization.

@bmarzeion for review

### CODE REVIEW

File: _includes/nav.html [hunk 1]
```diff
@@ -16,21 +16,21 @@
...
-                </li>
-                <li>
                     <a href="{{ site.baseurl }}/news/">News</a>
                 </li>
                 <li>
                     <a href="{{ site.baseurl }}/publications/">Publications</a>
                 </li>
                 <li>
-                    <a href="http://oggm.readthedocs.io" target="_blank">Documentation</a>
+                    <a href="{{ site.baseurl }}/oggmev/">OGGM e.V.</a>
```
Comment (bmarzeion) [inline _includes/nav.html]: perhaps rather "Organization" as the menu item?

File: oggmev/index.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+description: "The OGGM organization"
+header-img: "img/backgrounds/dahlbreen_v1.jpg"
+---
+
+**OGGM e.V.** is a non-profit organization based in Bremen, Germany.
+
+The purpose of the organization is the promotion of science and research
+in the field of climate and glaciology, in particular by promoting
+the development of an open source model for glaciers dynamics. This implies:
+- the organization and participation to events (conferences, workshops,
```
Comment (bmarzeion) [inline oggmev/index.md]: the organization of and participation in events...

File: oggmev/index.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+  seminars...) in relation to climate and cryospheric sciences
+- the publication of scientific or vulgarization material in peer-review
```
Comment (bmarzeion) [inline oggmev/index.md]: what is meant by "vulgarization"?

File: _includes/nav.html (hunk 1 above)
Comment (fmaussion) [reply _includes/nav.html]: Yes, agreed

File: oggmev/index.md (hunk 2 above)
Comment (fmaussion) [reply oggmev/index.md]: will correct

File: oggmev/index.md (hunk 3 above)
Comment (fmaussion) [reply oggmev/index.md]: Sorry, this is french for "popular science" (https://en.wikipedia.org/wiki/Popular_science). What about:

```
the publication of scientific or popular science material in ...
```

?

Comment (bmarzeion) [reply oggmev/index.md]: perhaps then: 
the preparation of technical and popular material for publication in peer-reviewed journals, newspapers, websites, social media, etc.

**Comment by fmaussion** [comment]
Thanks @bmarzeion !

**Pull request merged**